cmp/grid/Grid.ts

```diff
@@ -351,7 +352,7 @@ class GridLocalModel extends HoistModel {
 
             let childItems;
             if (!isEmpty(displaySpec.items)) {
-                const menu = new StoreContextMenu({items: displaySpec.items, gridModel: this.gridModel});
+                const menu = new StoreContextMenu({items: displaySpec.items, gridModel: this.model});
```

Comment:
Was this broken before?!

Reply:
maybe an uncommon code path?  Not sure, but was clearly a broken refactor.